qiskit/backends/aer/qasm_simulator.py

```diff
@@ -46,13 +48,16 @@ class QasmSimulator(BaseBackend):
     """C++ quantum circuit simulator with realistic noise"""
 
     DEFAULT_CONFIGURATION = {
-        'name': 'qasm_simulator',
+        'backend_name': 'qasm_simulator',
+        'backend_version': '1.0',
```

Comment:
The current .jsonschemas require the version to be in the form `X.Y.Z` (validates against `"[0-9]+.[0-9]+.[0-9]+$"`). I personally think we should make the schema more lenient, but in the meantime, would it make sense to stick to the current definition?